R/utils.R

```diff
@@ -84,26 +84,17 @@ h_identify_loq_values <- function(loqs_data) {
 h_caption_loqs_label <- function(loqs_data) {
   loq_values <- h_identify_loq_values(loqs_data)
 
-  if (is.na(loq_values$LLOQC)) {
-    lloq_value <- "NA"
-  } else {
-    lloq_value <- loq_values$LLOQC
-  }
-
-  if (is.na(loq_values$ULOQC)) {
-    uloq_value <- "NA"
-  } else {
-    uloq_value <- loq_values$ULOQC
-  }
+  loq_values$LLOQC <- ifelse(is.na(loq_values$LLOQC), "NA", loq_values$LLOQC)
+  loq_values$ULOQC <- ifelse(is.na(loq_values$ULOQC), "NA", loq_values$ULOQC)
```

Comment:
This will convert the LLOQC, ULOQC values from numbers to characters which I assume is what happened previously without ill effects

Reply:
okay. sounds good. thanks.